pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -32,7 +30,7 @@ type Client struct {
 }
 
 // NewClient creates new KUDO Client
-func NewClient(kubeConfigPath string, requestTimeout int64, validateInstall bool) (*Client, error) {
+func NewClient(kubeConfigPath string, requestTimeout int64) (*Client, error) {
```

Comment:
So the `NewClient` methods stays in `util/kudo` pkg and `ValidatedClient` is in `cilent` pgk? What is the reasoning of keeping these two in completely separated packages? As a new developer I would be surprised not to find these two in the same file.

P.S. This is not part of this PR (yet) but it is lamentable that somethings as central as `kudo.Client` lives in `util` pkg (which itself is an anti-pattern and code smell in Go)